tests/dist/src/test/java/org/hawkular/apm/tests/dist/AnalyticsServiceITest.java

```diff
@@ -108,7 +109,7 @@ public void initTest() {
     public void testGetUnboundEndpoints() throws Exception {
         Trace trace1 = new Trace();
         trace1.setId("1");
-        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
Well its your code (`// Within last hour`). I think you did it because then you can query data with default criteria (returns data for last hour)